stltovoxel.py

```diff
@@ -126,6 +98,7 @@ def file_choices(choices, fname):
         type=lambda s: file_choices(('.png', '.xyz', '.svx'), s),
         help='path to output files. The export data type is chosen by file extension. Possible are .png, .xyz and .svx')
     parser.add_argument('resolution', nargs='?', type=int, default=100, help='number of voxels in both directions')
+    parser.add_argument('pad', nargs='?', type=int, default=1, help='number of padding bits')
```

Comment:
number of padding pixels

Reply:
The pixel seems to be a more accurate representation.
Commit: 7fff41e888b51d76861a9453ec25fe2abe3b1f89